docs/kubernetes/KubernetesPolicy.md

```diff
@@ -28,14 +28,14 @@ Here is an example of how this looks in a pod spec.
 ...
   metadata:
     annotations:
-      policy: "allow tcp from label role=backend"
+      projectcalico.org/policy: "allow tcp from label role=backend"
```

Comment:
Should probably include a node indicating that pre-v0.2.0 versions of the plugin use "policy" as the key.